FoodBookApp/UI/Views/SpotDetail/ReviewsView.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  ReviewsView.swift
+//  FoodBookApp
+//
+//  Created by Laura Restrepo on 27/02/24.
+//
+
+import Foundation
+import SwiftUI
+
+
+struct ReviewsView: View {
+    @State private var model = ReviewsViewModel()
+    @State private var spotModel = SpotDetailViewModel()
+    let customGray = Color(red: 242/255, green: 242/255, blue: 242/255)
+    
+    var body: some View {
+            
+        // Header
+        VStack(spacing: 0){
+            ZStack(alignment: .leading) {
+                cancelButton
+                Text(spotModel.spot.name)
```

Comment:
This should be passed as a param to the ReviewView instead of using the SpotViewModel. In general, a View should have only 1 ViewModel. 

Take a look into passing params from parent to child component, I think it could just be a simple variable with which the ReviewsView is instantiated from within the parent once navigation is implemented.